www/js/diary/diaryHelper.ts

```diff
@@ -97,8 +97,8 @@ export function getBaseModeByText(text, labelOptions: LabelOptions) {
 export function isMultiDay(beginFmtTime: string, endFmtTime: string) {
   if (!beginFmtTime || !endFmtTime) return false;
   return (
-    moment.parseZone(beginFmtTime).format('YYYYMMDD') !=
-    moment.parseZone(endFmtTime).format('YYYYMMDD')
+    DateTime.fromISO(beginFmtTime).toFormat('YYYYMMDD') !=
+    DateTime.fromISO(endFmtTime).toFormat('YYYYMMDD')
```

Comment:
again, be careful with `parseZone` and make sure whatever you use from Luxon does the **exact** same thing